features/step_definitions/reek_steps.rb

```diff
@@ -11,69 +11,72 @@
 end
 
 Then /^it reports nothing$/ do
-  assert_exact_output '', all_stdout
+  expect(last_command_started).to have_output_on_stdout('')
```

Comment:
Hm, the above switch was what [the deprecation message said we should use](https://github.com/cucumber/aruba/blob/v0.9.0/lib/aruba/api/deprecated.rb#L560) and seems to work (and read) right to me…